src/main/java/core/basesyntax/impl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.CsvFileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvFileReaderServiceImpl implements CsvFileReaderService {
```

Comment:
```suggestion
public class FileReaderServiceImpl implements FileReaderService {
```
this service and interface has no specific logic for csv format